gslib/commands/perfdiag.py

```diff
@@ -235,13 +250,18 @@
 
 
 <B>NOTE</B>
-  The perfdiag command collects system information. It collects your IP address,
-  executes DNS queries to Google servers and collects the results, collects
-  network statistics information from the output of netstat -s, and looks at the
-  BIOS product name string. It will also attempt to connect to your proxy server
-  if you have one configured and will look up the location and storage class of
-  the bucket being used for performance testing. None of this information will
-  be sent to Google unless you choose to send it.
+  The ``perfdiag`` command runs a series of tests and collects system information,
+  such as the following: 
+  
+  + Records requester's IP address.
+  + Executes DNS queries to Google servers and collects the results.
+  + Collects network statistics information from the output of ``netstat -s`` and
+    evaluates the BIOS product name string.
+  + If configured, attempts to connect to the requester's proxy server
```

Comment:
>If configured, attempts to connect to the requester's proxy server

To me this sounds like, if the perdiag command itself is configured to connect to the proxy server, whereas old line `It will also attempt to connect to your proxy server if you have one configured` makes it sound like if the user has a proxy server configured, perfdiag will try to connect to it.